lib/plugins/base.rb

```diff
@@ -1,4 +1,4 @@
-require "#{Rails.root}/plugins/repository"
+require "#{Rails.root}/lib/plugins/repository"
```

Comment:
I'd have a global repo search for `:plugins`, I see some in `plugins.rake`, and `plugins/base.rb`